packages/core/src/__tests__/fixtures/.redocly.lint-ignore.yaml

```diff
@@ -0,0 +1,5 @@
+# This file instructs Redocly's linter to ignore the rules contained for specific parts of your API.
```

Comment:
No, it should not affect neighbours. Because of we make new Config in this test by mock and pass confiPath. It will cause only if 
- a) pass specific configPath to the `makeConfig` function in test
- b) place this ignore file in root - because by default we took `process.cwd()` path if no configPath passed